pom.xml

```diff
@@ -66,6 +66,7 @@
     <excludedGroups />
     <memoryMb>4096</memoryMb>
     <directMemoryMb>4096</directMemoryMb>
+    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
Do we need a task (new Jira) for refactoring the Drill java docs and moving onto Java 8 doclint?

Reply:
@vdiravka Please see DRILL-4547.

Reply:
Thanks